<?xml version="1.0"?>
<launch>

<arg name="sensor" value="lsm" />

<node name="imu_$(arg sensor)" pkg="navio2_ros" type="navio2_imu_node" respawn="true" output="screen">
  <param name="sensor_name" type="string" value="$(arg sensor)" />
  <param name="frame_id" value="imu_link" />
  <param name="sensor_frequency" value="50.0" />
</node>

</launch>
